core/serialize/serialize.cc

```diff
@@ -631,6 +631,7 @@ void SerializerImpl::pickle(Pickler &p, const Method &what) {
         pickle(p, a);
     }
     pickle(p, what.resultType);
+    p.putU4(what.intrinsicOffset);
```

Comment:
Whoops, good catch.

Reply:
I mentioned this in the commit message but I think this wasn't so much a bug as much as a choice not to serialize function pointers to a cache (before intrinsic became intrinsicOffset)